test/conftest.py

```
@@ -37,3 +43,107 @@
+streams:
+  test: "ffmpeg:{video_path}#loop"
+
+""")
+
+    # Start go2rtc
+    go2rtc_command = [
+        "./go2rtc",  # Adjust path to go2rtc executable if needed
+        "--config",
+        str(config_path),
+    ]
+    process = subprocess.Popen(go2rtc_command)
```

Ruff (0.11.9)

`subprocess` call: check for execution of untrusted input

(S603)